docs/management/api/trusted-apps-api.asciidoc

```diff
@@ -0,0 +1,557 @@
+:endpoint-artifact-list-id:     endpoint_trusted_apps
+:endpoint-artifact-name:        Trusted Applications
+:endpoint-artifact-name-single: Trusted Application
+[[trusted-applications-api]]
+=== {endpoint-artifact-name}
+
+Create, retrieve, update and delete endpoint <<trusted-apps-ov>> via API. {endpoint-artifact-name} are managed via the <<exceptions-api-overview>> using a static container id (`list_id`) of `pass:a[{endpoint-artifact-list-id}]` , which must be created prior to adding a trusted application. Access to these APIs require that a user have authorization to manage endpoints.
```

Comment:
The space before comma is unnecessary "`pass:a[{endpoint-artifact-list-id}]` ,"

Reply:
Ahh, yes, I'll correct it. I added a space because in my IDE I could not see it in the live rendered preview. I see that in the live docs, you can in fact see it, so I'll remove it